libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -868,6 +893,113 @@ public void startSwitcherActivityIfRequired() {
         }
 	}
 
+    /**
+     * Updates developer support features to match an Android Activity lifecycle
+     * event, including updating the app's authenticated Android Activity.
+     *
+     * @param authenticatedActivity The new authenticated Android Activity
+     *                              instance, or null
+     * @param lifecycleActivity     The Android Activity instance provided by
+     *                              the Activity lifecycle event
+     */
+    private void updateDeveloperSupportForActivityLifecycle(
```

Comment:
The display and hide of the developer support notification and dialog are handled here, which was a nice way to extract the logic from the Activity Lifecycle Callback object.